Description: Use source-date-epoch as timestamp source for documentation
Author: James Addison <jay+salsa@jp-hosting.net>
Origin: https://salsa.debian.org/js-team/npm/-/merge_requests/7
Forwarded: no
Reviewed-By: Yadd <yadd@debian.org>
Last-Update: 2022-04-24

--- a/docs/bin/docs-build.js
+++ b/docs/bin/docs-build.js
@@ -30,7 +30,8 @@
     .replace(/\[([^\]]+)\]\(\/using-npm\/([^)]+)\)/g, replacer)
     .trim()
 
-  await fs.writeFile(dest, marked(result), 'utf8')
+  const date = new Date(process.env.SOURCE_DATE_EPOCH)
+  await fs.writeFile(dest, marked(result, {date}), 'utf8')
 }
 
 run(...process.argv.slice(2)).catch((err) => {
